Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:16-alpine
```

Comment:
https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile have a look a this can just steal it

Reply:
sweet will use that